docs/route-tasks.md

```diff
@@ -0,0 +1,238 @@
+---
+id: route-tasks
+title: How to route Tasks
+sidebar_label: Route Tasks
+---
+
+## Overview
+
+This guide covers the recommended techniques to ensure that a Workflow and/or Activity will be executed by an appropriate Worker. In this guide we will look at two examples:
+
+1. A basic example that illustrates the fundamentals of routing a Task to a Worker.
+2. A more complex example involving Activities with host-specific file dependencies.
+
+Consider the specifications of a Worker:
+- A Worker only handles [Activity](activities) and [Workflow](workflows) Tasks that it is registered to handle. This means that a Task must be paired with a Worker that is registered to handle that Task.
+- A Worker runs on a specific host. This means that Activity Tasks that depend on host-specific resources, such as a local file, must be executed by a Worker on that host.
```

Comment:
Actually a worker runs in a specific process. And it is possible for an activity to depend on process-specific resources like in-memory cache.